src/styles/settings/calendar.scss

```diff
@@ -0,0 +1,59 @@
+$day-size: 100px;
+$calendar-padding: 82px;
+$calendar-size: ($day-size * 7) + (2 * $calendar-padding) + 6;
+$bg-color: #eee;
+$days: (
+  mon: 0,
+  tue: $day-size + 1,
+  wed: 2 * $day-size + 2,
+  thu: 3 * $day-size + 3,
+  fri: 4 * $day-size + 4,
+  sat: 5 * $day-size + 5,
+  sun: 6 * $day-size + 6,
+);
+$month-lengh: 28, 29, 30, 31;
```

Comment:
There is a typo in the variable name `$month-lengh`. It should be `$month-length` to correctly represent the concept of month length.